compact/definitions.xml

```diff
@@ -567,11 +567,11 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
 
     <constant name="HcalBarrelStavesN"          value="12"/>
     <constant name="HcalEndcapP_zmin"     value="EcalEndcapP_zmin + EcalEndcapP_length"/>
-    <constant name="HcalEndcapP_length"   value="120.0*cm"/>
+    <constant name="HcalEndcapP_length"   value="130.0*cm"/>
```

Comment:
Why 130 cm when the https://eic.jlab.org/Geometry/Detector/Detector-20220929172703.html says 140 cm? Are we sticking to that detector parameter table or not?

Reply:
I brought this up during the calorimetry meeting. The detector is 130 cm + 10 cm of electronics. I have not included electronics here and was told this was fine

Reply:
I can change the length in `definitions.xml` to equal the _actual_ length of 140 cm, and then make the hcal only actually 130 cm, perhaps. This would solve other definitions relying on `HcalEndcapP_length` in `definitions.xml`

Reply:
But essentially since this isn't actually my detector and I'm just helping out ORNL, I have no idea what the electronics system is like nor what I should implement for it

Reply:
Yeah, I think that would be better. Thanks for the clarification!

Reply:
Alright, I'll adjust that it in a bit!